src/config/passport.js

```diff
@@ -0,0 +1,21 @@
+import passport from 'passport';
+import GoogleStrategy from 'passport-google-oauth20';
+import FacebookStrategy from 'passport-facebook';
+import * as constant from './constants';
+
+passport.use(new GoogleStrategy({
```

Comment:
You have done good work bro, but why don't you consider abstracting  the relevant strategies config and then import